src/components/Filters/index.tsx

```diff
@@ -81,291 +89,217 @@ const FilterWrapper = styled(Box)`
 	position: relative;
 `;
 
+interface EditModel {
```

Comment:
This is identical to `FilterSignature` and we are cross-assigning vars with casting. Should we dedup them or make the one extend the other?